src/index.html

```diff
@@ -3,11 +3,286 @@
+              </div>
+            </div>
+
+            <div><p class="gallery__caption">Headphones & earbuds</p></div>
+
+            <div class="gallery__row">
+              <div class="
+              gallery__photo
+              gallery__photo--d-1-8
+              gallery__photo--t-1-4
+              "
+              >
+                <img
+                  src="./images/photos/gallery_3.png"
+                  alt="Gallery 3"
+                  class="gallery__image"
+                >
+              </div>
+              <div class="
+              gallery__photo
+              gallery__photo--d-9-12
+              gallery__photo--t-5-6
+              ">
+                <img
+                  src="./images/photos/gallery_4.png"
+                  alt="Gallery 4"
+                  class="gallery__image"
+                >
+              </div>
+            </div>
+
+            <div><p class="gallery__caption">Speakers</p></div>
+
+            <div class="gallery__row">
+              <div class="
+              gallery__photo
+              gallery__photo--d-1-4
+              gallery__photo--t-1-2
+              "
+            >
+                <img
+                  src="./images/photos/gallery_5.png"
+                  alt="Gallery 5"
+                  class="gallery__image"
+                >
+              </div>
+              <div class="
+              gallery__photo
+              gallery__photo--d-5-12
+              gallery__photo--t-3-6
+              ">
+                <img
+                  src="./images/photos/gallery_6.png"
+                  alt="Gallery 6"
+                  class="gallery__image"
+                >
+              </div>
+            </div>
+
+            <div><p class="gallery__caption">Audio sunglasses</p></div>
+
+          </section>
+          <a href="#" class="gallery__button button">View all</a>
+        </div>
+      </section>
+
+      <section class="direct">
+        <div class="direct__content-wrapper">
+          <div class="direct__title-wrapper">
+            <h2 class="direct__title">Why buy direct from Bose</h2>
+          </div>
+          <div class="direct__content">
+            <p class="direct__highlighted">
+              Free 2-day shipping and returns<br>
+              90-day risk-free trial<br>
+              World class customer service<br>
+              My Bose account management<br>
+            </p>
+            <p class="direct__text">
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact">
+        <div class="contact__content-wrapper">
+          <h2 class="contact__title">Contact us</h2>
+          <div class="contact__form-wrapper">
+            <form
```

Comment:
fields validation required - when trying to submit a form - validation helpers should show up:
![form fields validation](https://user-images.githubusercontent.com/39308717/162903087-fbafcdae-5487-4b12-8e71-84e8979b3bb5.png)
> 13.	Placeholders in the forms suggest what to enter; apply validation of the form fields (required, email / tel etc.), then it is clear in what format to enter the data